src/functionalTest/resources/features/F-205 - Validate NoC Questions/F-205.feature

```diff
@@ -16,6 +16,7 @@ Feature: F-205 Validate Notice of Change Answers
       And a case [created by Richard - C1 - on behalf of Mario as the applicant which is auto-assigned to Richard's organisation] created as in [204_Case_Creation],
       And a successful call [to get the update org policy event trigger] as in [204_Update_Org_Policies_Token_Creation],
       And a successful call [to update the applicant org policy] as in [204_Update_Org_Policies],
+      And a wait time of [15] seconds [to allow for the cache to refresh],
```

Comment:
You could describe this wait time in more none techie way...

like wait time for latest xyz definition data to take effect for abc function

Reply:
Yeah good idea, will take a look at it 👍